roboteam_ai/src/utilities/RobotDealer.cpp

```diff
@@ -367,6 +365,14 @@ void RobotDealer::setUseSeparateKeeper(bool useSeparateKeeper) {
     RobotDealer::useSeparateKeeper = useSeparateKeeper;
 }
 
+bool RobotDealer::keeperExistsInWorld() {
+    auto us = world::world->getUs();
+
+    return std::find_if(us.begin(), us.end(), [](world::Robot & robot) {
+        return robot.id == getKeeperID();
+    }) != us.end();
```

Comment:
I think this really doesn't need to be a lambda since not many people this year or next will be proficient in functional style programming